themes/app/templates/Includes/Breadcrumb.ss

```diff
@@ -0,0 +1,5 @@
+<div class="breadcrumb">
```

Comment:
I dont think this file is necessary you can take the contents of this file and put it all into `BreadcrumbsTemplate.ss` 

Then on `Page.ss` you can just call `$Breadcrumbs` as `$Breadcrumbs` shhould render `BreadcrumbsTemplate.ss`